### PR TITLE
Replace unsupported characters to have working stdout print.

### PR DESCRIPTION
**Pull request by goodov**
Related https://github.com/brave/brave-browser/issues/19063

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: script/lib/util.py [hunk 1]
```diff
@@ -183,15 +183,18 @@ def execute(argv, env=os.environ):  # pylint: disable=dangerous-default-value
...
             # Print the output instead of raising it, so that we get pretty output.
             # Most useful erroroutput from typescript / webpack is in stdout
             # and not stderr.
-            print(stdout)
-            raise RuntimeError('Command \'%s\' failed' %
-                               (' '.join(argv)), stderr)
+            print(printable_stdout)
+            if process.returncode != 0:
+                raise RuntimeError('Command \'%s\' failed' % (' '.join(argv)),
+                                   stderr)
```
Comment (petemill) [inline script/lib/util.py:197]: We're not doing the same safe encoding for `stderr` - perhaps during the `raise` python internally does the correct encoding? This is the only place chromium does their "logging" (and they also don't do the extra encoding) so perhaps that's an ok assumption.

**Review by petemill — APPROVED** [review]

**Review by mherrmann — APPROVED** [review]

**Comment by mihaiplesa** [comment]
Closing in favour of https://github.com/brave/devops/pull/6749

**Comment by goodov** [comment]
I'd prefer this to be merged in case some other process will decide to spit out unsupported chars into our python runner on win32.

File: script/lib/util.py (hunk 1 above)
Comment (goodov) [reply script/lib/util.py:197]: I checked and yes, it prints unsupported chars as `\u...` in the exception handler.

**Pull request merged**